utils/chem/chem_diagb.h

```diff
@@ -230,15 +236,66 @@ namespace gdasapp {
         }
       }
 
+      std::cout << "Number of variables in chemVars: " << chemVars.size() << std::endl;
+
+      for (size_t i = 0; i < chemVars.size(); ++i) {
+        std::cout << "Variable " << i << ": " << chemVars[i].name() << std::endl;
+}
+
       // Rescale
+      //
       if (fullConfig.has("rescale")) {
-        double rescale;
-        fullConfig.get("rescale", rescale);
-        util::multiplyFieldSet(bkgErrFs, rescale);
-      }
+    // Define rescaling factors for each species
+        std::vector<double> rescaleFactors = {.5, 1.0, 1.0, 1.0, 1.0, 2.0, 2.0, 2.0, 2.0, 2.0, 1.0, 1.0, 1.0, 1.0};
```

Comment:
This should be read in from the YAML that way we can test/tune without needing to recompile

Reply:
Never mind, I didn't read, you said that in the description :-)

Reply:
Yes, I'm working on that next.